src/main/java/use_case/probability/stand/ProbabilityStandInteractor.java

```diff
@@ -0,0 +1,237 @@
+     * @return an int representing the floor of the percentage of winning scenarios userDeck beats dealerDeck.
+     */
+    private int winPercentage(int userScore) {
+
+        Map<String, Integer> winScenario = this.countWinsandGames(userScore, dealerCards, unknownCards);
+
+        return Math.floorDiv(winScenario.get(WINS), winScenario.get(SCENARIO));
+    }
+
+    /**
+     * Actively counts the amount of times userDeck wins a game and builds ALL possible decks the dealer can feasibly
+     * have knowing that the dealer will only ever pull another card from the deck if their score is less than 17.
+     * To take care of a very unlikely edge case, if the deck of cards runs out before the dealer hits 21, this is
+     * automatically counted as a player win.
+     * @param userScore an integer representing the score of the player's hand.
+     * @param currentDealer a map containing all cards dealer theoretically has at time of call.
+     * @param unknownDeck a map containing what cards are assumed to be unknown.
+     * @return a map that contains a key representing how many scenarios have been tested and in how many of them the
+     * player wins.
+     */
+    private Map<String, Integer> countWinsandGames(int userScore, Map<Integer, Integer> currentDealer,
+                                                   Map<Integer, Integer> unknownDeck) {
+
+        int playerWin = 0;
+        int scenarioCount = 0;
+        Map<String, Integer> result = new HashMap<>();
+
+        result.put(SCENARIO, 0);
+        result.put(WINS, 0);
+
+        for (int i = 1; i <= 11; i++) {
+
+            // check if there is at least one card of this value left in the unknown card list, if there is not then
+            // continue to the next value to remove useless iterations.
+            if (unknownDeck.get(i) < 1) {
+                continue;
+            }
+
+            Map<Integer, Integer> conditionalDealer = new HashMap<>(currentDealer);
+            conditionalDealer.compute(i, (k, v) -> v + 1);
+
+
+            int dealerScore = handScore(conditionalDealer);
+
+            if (dealerScore > BLACKJACK) {
+                playerWin += unknownDeck.get(i);
+                scenarioCount += unknownDeck.get(i);
+            }
+
+            // dealerScore will not result in a bust.
+            else if (dealerScore >= 17) {
+
+                if (dealerScore < userScore) {
+                    playerWin += unknownDeck.get(i);
+                }
+                scenarioCount += unknownDeck.get(i);
+            }
+
+            else  {
+
+                Map<Integer, Integer> conditionalUnknown = new HashMap<>(unknownDeck);
+                conditionalUnknown.compute(i, (k, v) -> v - 1);
+
+                // edge-case that the deck is empty to prevent infinite recursion
+                if (this.sumCollection(conditionalUnknown.values()) == 0) {
+                    playerWin++;
```

Comment:
I could be wrong but I think here you need to compare the player and dealer scores first, since I think it's still possible the dealer could beat the player (e.g. if the player stands on 15 and the dealer has 16)

Reply:
Have implemented this for my next commit!